src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -505,23 +504,15 @@ def _check_and_init_precision(self) -> PrecisionPlugin:
 
             if isinstance(self.accelerator, HPUAccelerator):
                 return HPUPrecisionPlugin(self._precision_flag)
-        if isinstance(self.accelerator, XLAAccelerator):
-            if self._precision_flag == "32-true":
-                return XLAPrecisionPlugin()
-            if self._precision_flag in ("16-mixed", "bf16-mixed"):
-                if self._precision_flag == "16-mixed":
-                    rank_zero_warn(
-                        "You passed `Trainer(accelerator='tpu', precision='16-mixed')` but AMP with fp16"
-                        " is not supported on TPUs. Using `precision='bf16-mixed'` instead."
-                    )
```

Comment:
TODO: add this back?

Reply:
Yes, I deleted these lines because we have an error already in the xla precision plugin. Would you like a have a customized error message like the warning above specifically for "-mixed"?